app/javascript/src/NewService/components/NewServiceForm.jsx

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+import React, {useState, useEffect} from 'react'
+import {fetch as fetchPolyfill} from 'whatwg-fetch'
+import 'core-js/es6/promise'
+import {ServiceSourceForm, FormTemplate} from 'NewService'
+import type {SourceFormProps, FormTemplateProps} from 'NewService/types'
+
+const BASE_URL = '/p/admin/service_discovery/'
```

Comment:
Small thing, but I think it's better omitting the leading slash here and repeating it wherever necessary. Reading the interpolation `${BASE_URL}something` is weird.

Reply:
I'd say it should not be called URL as it's not. It's a path.